### PR TITLE
feat(replays): Better console messages + add string formatting

### PR DESCRIPTION
**Pull request by billyvg**
This allows console messages to be formatted properly using sprintf-js. Also adds some color to warning/error level messages.

New:
image

Compared to existing:
image

### CODE REVIEW

**Comment by billyvg** [comment]
There are issues formatting when the first arg is not a string.

**Comment by Jesse-Box** [comment]
This is nice, what opportunities do we have @billyvg to style this?

**Comment by billyvg** [comment]
@Jesse-Box let's do it, I just didn't want to commit to much to it before the demo

File: static/app/views/replays/detail/console/consoleMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  // TODO(replays): Add better support for AnnotatedText (e.g. we use message
+  // args from breadcrumb.data.arguments and not breadcrumb.message directly)
+  return (
+    <Fragment>
+      <AnnotatedText
+        meta={getMeta(breadcrumb, 'message')}
+        value={[formattedMessage, ...restArgs].join(' ')}
+      />
+    </Fragment>
+  );
```
Comment (ryan953) [inline static/app/views/replays/detail/console/consoleMessage.tsx:63]: bonus `<Fragment>`? i'm guessing at what eslint will say

```suggestion
  return (
    <AnnotatedText
      meta={getMeta(breadcrumb, 'message')}
      value={[formattedMessage, ...restArgs].join(' ')}
    />
  );
```

File: static/app/views/replays/detail/console/index.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const Console = styled(BaseConsole)`
+  font-family: ${p => p.theme.text.familyMono};
+  font-size: 0.8em;
+`;
+
+const ConsoleTable = styled(Panel)`
+  display: grid;
+  grid-template-columns: max-content auto;
+`;
```
Comment (ryan953) [inline static/app/views/replays/detail/console/index.tsx:37]: we should be able to drop `className?: string` from the props if we combine the css here. i think all the css is being applied to the Panel now already. 

```
const ConsoleTable = styled(Panel)`
  font-family: ${p => p.theme.text.familyMono};
  font-size: 0.8em;
  display: grid;
  grid-template-columns: max-content auto;
`;
```

**Review by ryan953 — APPROVED** [review]
:shipit: 

css nits & eslint notwithstanding

File: static/app/views/replays/detail/console/index.tsx (hunk 2 above)
Comment (billyvg) [reply static/app/views/replays/detail/console/index.tsx]: Oops yeah I forgot to clean-up a bunch of stuff, thanks!

**Pull request merged**